src/krux/qr.py

```diff
@@ -88,7 +95,7 @@ def total_count(self):
             # Single-part URs have no expected part indexes
             if self.decoder.fountain_decoder.expected_part_indexes is None:
                 return 1
-            return self.decoder.expected_part_count()
+            return self.decoder.expected_part_count() * 2
```

Comment:
Where `*2` comes from? 
`fountain_decoder.py` has an `estimated_percent_complete()` which uses:
`estimated_input_parts = self.expected_part_count() * 1.75`

Reply:
The 1.75 is an estimative they use, but it is wrong! In the end the scan will only finish after reading X estimated parts + X index parts... so it is 2 times the estimated parts